concourse/pipelines/test-certificate-rotation.yml

```diff
@@ -126,6 +126,44 @@ jobs:
                     echo "timeout waiting for app-availability-tests job to start"
                     exit 1
 
+          - task: wait-for-c2c-availability-tests
```

Comment:
Is it possible to refactor this in to a task file?

Reply:
It could be made a task file on its own, perhaps, I originally tried to make it a task file shared between `create-cloudfoundry` and `test-certificate-rotation` but there is an input step.

We could make the input step optional and parameterise the if statement:

```
# currently
                HEALTHCHECK_DEPLOYED=$(cat deployed-healthcheck/healthcheck-deployed)
                if [ "${HEALTHCHECK_DEPLOYED}" = "no" ]; then
                  echo "Skipping wait-for-c2c-availability-tests because the healthcheck is not deployed"
                  exit 0
                fi
```

Reply:
resolved in 65cbe38 and 4735dec